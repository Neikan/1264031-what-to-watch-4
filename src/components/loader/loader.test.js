// Импорт библиотек
import React from "react";
import renderer from "react-test-renderer";

// Импорт компонентов
import Loader from "./loader.jsx";


describe(`Test Loader component`, () => {
  test(`Loader component is created and rendered correctly`, () => {
    const tree = renderer.create(
        <Loader />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
