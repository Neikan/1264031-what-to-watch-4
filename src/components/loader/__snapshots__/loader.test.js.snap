// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Loader component Loader component is created and rendered correctly 1`] = `
<section>
  <h2
    className="loader"
  >
    Loading... Please wait
  </h2>
</section>
`;
